Software Engineer iOS.md

```diff
@@ -29,21 +29,14 @@
 
 ## Apply
 
-If you happen to be a pragmatic, test-driven software craftsman or craftswoman who likes working on the edge of technology, come and help our outstanding, truly agile development teams in [Munich, DE](https://goo.gl/maps/2KKGh), in [Bottighofen, CH](https://goo.gl/maps/X7bZ3), in [Poznan, PL](https://goo.gl/maps/AiHKJ), or in [Amsterdam, NL](https://goo.gl/maps/AJHpM3yYUzL2).
+If you happen to be a pragmatic, test-driven software craftsman or craftswoman who likes working on the edge of technology, come and help our outstanding, truly agile development team at Lake Constance in [Bottighofen, CH](https://goo.gl/maps/X7bZ3).
```

Comment:
Instead of craftsmen & craftswoman which excludes people who doesn’t see themselves as man or woman, you could use Software Crafter. We use the same wording on https://apprenticeship.holidaycheck.com

Reply:
Thanks for the hint. I pushed a new commit which tackles this issue.